### PR TITLE
[READY] Adds new easter egg for sec access

### PR DESCRIPTION
**Pull request by JohnGinnane**
:cl: JohnGinnane
add: People with sec access now have a 2% chance to hear an easter egg when opening doors
/:cl:

2% chance for a meme sound to play

### CODE REVIEW

**Comment by ghost** [comment]
+1

**Comment by Carbonhell** [comment]
Why so many easter eggs just for doors? This feels weird

**Comment by JohnGinnane** [comment]
\>2 sounds
\>"so many"
:thinking:

**Comment by Eagle-Eyess** [comment]
+1

**Comment by HippieBartels** [comment]
+1

**Comment by Carbonhell** [comment]
Ready?

**Comment by JohnGinnane** [comment]
Yep

File: hippiestation/code/game/machinery/doors/airlock.dm [hunk 1]
```diff
@@ -28,6 +28,28 @@
...
+			var/list/access = H.wear_id.GetAccess()
+			if(ACCESS_SEC_DOORS in access)
+				if (prob(EASTEREGG_CHANCE) && !easteregg_triggered)
+					doorOpen = 'hippiestation/sound/machines/airlockopen_sec.ogg'
+					easteregg_triggered = TRUE
+	. = ..()
+
+/obj/machinery/door/airlock/attack_hand(mob/user)
+	var/mob/living/carbon/human/H = user
+	if (H)
```
Comment (Carbonhell) [inline hippiestation/code/game/machinery/doors/airlock.dm:44]: No istype? Also this copypasta is ugly man

**Review by Carbonhell — CHANGES_REQUESTED** [review]
Copypasta

File: hippiestation/code/game/machinery/doors/airlock.dm (hunk 1 above)
Comment (JohnGinnane) [reply hippiestation/code/game/machinery/doors/airlock.dm:44]: it's not copy pasta ;_;

File: hippiestation/code/game/machinery/doors/airlock.dm [hunk 2]
```diff
@@ -28,6 +28,28 @@
 
 	return .
 
+/obj/machinery/door/airlock/bumpopen(mob/living/user)
+	var/mob/living/carbon/human/H = user
+	if (istype(H) && H)
```
Comment (yoyobatty) [inline hippiestation/code/game/machinery/doors/airlock.dm:33]: Maybe ishuman works better

Comment (Carbonhell) [reply hippiestation/code/game/machinery/doors/airlock.dm:33]: checking H is useless if you also istype, istype returns false if the argument is null

File: hippiestation/code/game/machinery/doors/airlock.dm (hunk 1 above)
Comment (Carbonhell) [reply hippiestation/code/game/machinery/doors/airlock.dm:44]: alright

**Review by Carbonhell — CHANGES_REQUESTED** [review]

**Comment by Carbonhell** [comment]
Stale...

**Pull request closed without merging**